geopandas/base.py

```diff
@@ -964,6 +964,39 @@ def total_bounds(self):
     def sindex(self):
         return self.geometry.values.sindex
 
+    @property
+    def has_sindex(self):
+        """Check the existence of the spatial index without generating it.
+
+        Use the `.sindex` attribute on a GeoDataFrame or GeoSeries
+        to generate a spatial index if it does not yet exist,
+        which may take considerable time based on the underlying index
+        implementation.  This function quickly checks
```

Comment:
No longer necessary: `This function quickly checks...`

Reply:
Donzo.